docs/buildah-run.md

```diff
@@ -0,0 +1,37 @@
+## buildah-run "March 2017"
+
+## NAME
+buildah run - Run a command inside of the container. 
+
+
+## SYNOPSIS
+**buildah** **run** **containerID** [*command options* [...]] 
+
+## DESCRIPTION
+Runs a specified command using the container's root filesystem as a root filesystem, using configuration settings inherited from the container's image or as specified using previous calls to the config command
```

Comment:
This actually launches a container for the running of the command, so that it is somewhat safer then running a random command.  THe command runs inside of the container.